// Copyright 2023 Signal Messenger, LLC
// SPDX-License-Identifier: AGPL-3.0-only

syntax = "proto3";

package svr2.tests;
option optimize_for = LITE_RUNTIME;

message SimplePB {
  string str = 1;
}
